Gemfile.lock

```diff
@@ -477,6 +483,7 @@ GEM
 
 PLATFORMS
   ruby
+  universal-mingw32
```

Comment:
From `v17.10.3` tag:
```
  arm64-darwin-21
  ruby
  x64-mingw32
  x86-mingw32
  ```
I think we at least need one of the `{architecture}-mingw32` added... `x64-mingw32`.